src/views/pages/tasks/tasks-page.js

```diff
@@ -420,12 +455,14 @@ export class TasksPage extends Component {
     return (
       <div>
         <div className="g-col">
-          { <TaskFilters
-            filter = { this.props.filterType }
-            selectedProject = { this.props.selectedProject }
-            projectUrl = { projectUrl } //TODO - should be from state
-            labels = { this.state.labelPool }
-            onLabelChange = { this.onLabelChanged }
+          {<TaskFilters
```

Comment:
No need to wrap it with `{}` unless there is a render-condition for instance: `{isVisible && <Component/>}`